devops/pipeline/stages/binary_signing.yaml

```diff
@@ -0,0 +1,112 @@
+# Binary signing stage (MSFT Internal)
```

Comment:
>(MSFT Internal) [](http://example.com/codeflow?start=23&length=15)

Is this accurate? If so, should this YAML be here?

Reply:
I recommend we have this file in here. The MSFT Internal tag is just used to show that its not meant for public consumption. Maybe a better tag should be used?